src/leds/README.md

```diff
@@ -0,0 +1,84 @@
+# Leds application
+
+The `Leds` application demonstrates how to read and write to the leds device file to control the physical leds on a device.
```

Comment:
s/led/LED/g

=> "The LED Application demonstrates how to control physical LEDs shared by the host system.